understanding/21/non-text-contrast.html

```diff
@@ -388,6 +388,16 @@ <h4>Infographics</h4>
 				<p>An infographic can use text which meets the other criteria to minimise the number of graphical objects required for understanding. For example, using text with sufficient contrast to provide the values in a chart. A long description would also be sufficient because then the infograph is not relied upon for understanding.</p>
 
 				</section>
+				
+				<section>
+					<h4>Symbolic text characters</h4>
+					<p>When text characters are used as symbols – used for their visual appearance, rather than <q>expressing something in human language</q> – they fall under the definition of <a>non-text content</a>.</p>
+					<figure id="figure-text-symbols">
+						<img alt="A button using an uppercase 'X' and a button with a greater-than character" src="img/buttons-text-symbols.png" />
+						<figcaption>Even though the two buttons use text characters — an uppercase <q>X</q>, often used for "Close" buttons, and a <q>&gt;</q> character, to act as a right-pointing arrow — they count as non-text characters/symbols. Their contrast ratio of just above 3:1 passes this Success Criterion.</figcaption>
```

Comment:
Don't want to complicate things - but is a greater-than character really something that "expresses something in human language"? Is it not a mathematical symbol? One possible way to edit this is to say: 

> "When text characters like "x" or "i", or mathematical symbols like ">", are used as symbolic images, that is, used for their visual appearance, rather than <q>expressing something in human language or in a mathematic formula</q> – they fall under the definition of <a>non-text content."

I can live with it as it is as well, though.